Server version need to update to 0.11.1

**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
self = <test_ping.TestPing object at 0x7f05690f52e8>
connect = <milvus.client.stub.Milvus object at 0x7f0569136dd8>

    def test_server_version(self, connect):
        '''
        target: test get the server version
        method: call the server_version method after connected
        expected: version should be the milvus version
        '''
        res = connect.server_version()
>       assert res == __version__
E       AssertionError: assert '0.11.0' == '0.11.1'
E         - 0.11.0
E         ?      ^
E         + 0.11.1
```

**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Method of installation**
- [x] Docker/cpu
- [x] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)


- Milvus version (master or released version)
 - 0.11.1

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/ci/jenkins/Jenkinsfile
+++ b/ci/jenkins/Jenkinsfile
@@ -2,7 +2,7 @@
 @Library('mpl') _
 
 String cron_timezone = "TZ=Asia/Shanghai"
-String cron_string = BRANCH_NAME == "0.11.1" ? "50 3 * * * " : ""
+String cron_string = BRANCH_NAME == "0.11.1" ? "30 0 * * * " : ""
 
 pipeline {
     agent none

--- a/ci/jenkins/Jenkinsfile
+++ b/ci/jenkins/Jenkinsfile
@@ -2,7 +2,7 @@
 @Library('mpl') _
 
 String cron_timezone = "TZ=Asia/Shanghai"
-String cron_string = BRANCH_NAME == "master" ? "50 3 * * * " : ""
+String cron_string = BRANCH_NAME == "0.11.1" ? "50 3 * * * " : ""
 
 pipeline {
     agent none

--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -26,7 +26,7 @@ get_build_type( TARGET  BUILD_TYPE
 message( STATUS "Build type = ${BUILD_TYPE}" )
 
 get_milvus_version( TARGET  MILVUS_VERSION 
-                            DEFAULT "0.11.0" )
+                            DEFAULT "0.11.1" )
 message( STATUS "Build version = ${MILVUS_VERSION}" )
 
 get_last_commit_id( LAST_COMMIT_ID )

--- a/tests/milvus_python_test/entity/test_search.py
+++ b/tests/milvus_python_test/entity/test_search.py
@@ -859,23 +859,24 @@ class TestSearchBase:
                 assert res[i]._distances[0] < epsilon
                 assert res[i]._distances[1] > epsilon
 
-    def _test_query_entities_with_field_less_than_top_k(self, connect, collection):
+    def test_query_entities_with_field_less_than_top_k(self, connect, id_collection):
         """
         target: test search with field, and let return entities less than topk
         method: insert entities and build ivf_ index, and search with field, n_probe=1
         expected:
         """
-        entities, ids = init_data(connect, collection)
-        simple_index = {"index_type": "IVF_SQ8", "params": {"nlist": 200}, "metric_type": "L2"}
-        connect.create_index(collection, field_name, simple_index)
-        default_query, default_query_vecs = gen_query_vectors(field_name, entities, default_top_k, nq, search_params={"nprobe": 1})
+        entities, ids = init_data(connect, id_collection, auto_id=False)
+        simple_index = {"index_type": "IVF_FLAT", "params": {"nlist": 200}, "metric_type": "L2"}
+        connect.create_index(id_collection, field_name, simple_index)
+        # logging.getLogger().info(connect.get_collection_info(id_collection))
+        top_k = 300
+        default_query, default_query_vecs = gen_query_vectors(field_name, entities, top_k, nq, search_params={"nprobe": 1})
         expr = {"must": [gen_default_vector_expr(default_query)]}
         query = update_query_expr(default_query, expr=expr)
-        res = connect.search(collection, query, fields=["int64"])
-        for r in res[0]:
-            logging.getLogger().info(getattr(r.entity, "int64"))
+        res = connect.search(id_collection, query, fields=["int64"])
         assert len(res) == nq
-        assert len(res[0]) == default_top_k
+        for r in res[0]:
+            assert getattr(r.entity, "int64") == getattr(r.entity, "id")
 
 
 class TestSearchDSL(object):
@@ -1012,6 +1013,11 @@ class TestSearchDSL(object):
         res = connect.search(collection, query)
         assert len(res) == nq
         assert len(res[0]) == default_top_k
+        limit = default_nb // 2
+        for i in range(nq):
+            for result in res[i]:
+                logging.getLogger().info(result.id)
+                assert result.id in ids[:limit]
         # TODO:
 
     def test_query_term_values_parts_in(self, connect, collection):

--- a/tests/milvus_python_test/entity/test_search.py
+++ b/tests/milvus_python_test/entity/test_search.py
@@ -22,7 +22,9 @@ entity = gen_entities(1, is_normal=True)
 entities = gen_entities(default_nb, is_normal=True)
 raw_vectors, binary_entities = gen_binary_entities(default_nb)
 default_query, default_query_vecs = gen_query_vectors(field_name, entities, default_top_k, nq)
-default_binary_query, default_binary_query_vecs = gen_query_vectors(binary_field_name, binary_entities, default_top_k, nq)
+default_binary_query, default_binary_query_vecs = gen_query_vectors(binary_field_name, binary_entities, default_top_k,
+                                                                    nq)
+
 
 def init_data(connect, collection, nb=1200, partition_tags=None, auto_id=True):
     '''
@@ -541,8 +543,10 @@ class TestSearchBase:
         nq = 2
         search_param = {"nprobe": 1}
         entities, ids = init_data(connect, collection, nb=nq)
-        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True, search_params=search_param)
-        inside_query, inside_vecs = gen_query_vectors(field_name, entities, default_top_k, nq, search_params=search_param)
+        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True,
+                                        search_params=search_param)
+        inside_query, inside_vecs = gen_query_vectors(field_name, entities, default_top_k, nq,
+                                                      search_params=search_param)
         distance_0 = l2(vecs[0], inside_vecs[0])
         distance_1 = l2(vecs[0], inside_vecs[1])
         res = connect.search(collection, query)
@@ -559,7 +563,8 @@ class TestSearchBase:
         entities, ids = init_data(connect, id_collection, auto_id=False)
         connect.create_index(id_collection, field_name, get_simple_index)
         search_param = get_search_param(index_type)
-        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True, search_params=search_param)
+        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True,
+                                        search_params=search_param)
         inside_vecs = entities[-1]["values"]
         min_distance = 1.0
         min_id = None
@@ -587,9 +592,11 @@ class TestSearchBase:
         metirc_type = "IP"
         search_param = {"nprobe": 1}
         entities, ids = init_data(connect, collection, nb=nq)
-        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True, metric_type=metirc_type,
+        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True,
+                                        metric_type=metirc_type,
                                         search_params=search_param)
-        inside_query, inside_vecs = gen_query_vectors(field_name, entities, default_top_k, nq, search_params=search_param)
+        inside_query, inside_vecs = gen_query_vectors(field_name, entities, default_top_k, nq,
+                                                      search_params=search_param)
         distance_0 = ip(vecs[0], inside_vecs[0])
         distance_1 = ip(vecs[0], inside_vecs[1])
         res = connect.search(collection, query)
@@ -608,7 +615,8 @@ class TestSearchBase:
         get_simple_index["metric_type"] = metirc_type
         connect.create_index(id_collection, field_name, get_simple_index)
         search_param = get_search_param(index_type)
-        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True, metric_type=metirc_type,
+        query, vecs = gen_query_vectors(field_name, entities, default_top_k, nq, rand_vector=True,
+                                        metric_type=metirc_type,
                                         search_params=search_param)
         inside_vecs = entities[-1]["values"]
         max_distance = 0
@@ -685,7 +693,8 @@ class TestSearchBase:
         query_int_vectors, query_entities, tmp_ids = init_binary_data(connect, binary_collection, nb=1, insert=False)
         distance_0 = substructure(query_int_vectors[0], int_vectors[0])
         distance_1 = substructure(query_int_vectors[0], int_vectors[1])
-        query, vecs = gen_query_vectors(binary_field_name, query_entities, default_top_k, nq, metric_type="SUBSTRUCTURE")
+        query, vecs = gen_query_vectors(binary_field_name, query_entities, default_top_k, nq,
+                                        metric_type="SUBSTRUCTURE")
         res = connect.search(binary_collection, query)
         assert len(res[0]) == 0
 
@@ -699,7 +708,8 @@ class TestSearchBase:
         top_k = 3
         int_vectors, entities, ids = init_binary_data(connect, binary_collection, nb=2)
         query_int_vectors, query_vecs = gen_binary_sub_vectors(int_vectors, 2)
-        query, vecs = gen_query_vectors(binary_field_name, entities, top_k, nq, metric_type="SUBSTRUCTURE", replace_vecs=query_vecs)
+        query, vecs = gen_query_vectors(binary_field_name, entities, top_k, nq, metric_type="SUBSTRUCTURE",
+                                        replace_vecs=query_vecs)
         res = connect.search(binary_collection, query)
         assert res[0][0].distance <= epsilon
         assert res[0][0].id == ids[0]
@@ -718,7 +728,8 @@ class TestSearchBase:
         query_int_vectors, query_entities, tmp_ids = init_binary_data(connect, binary_collection, nb=1, insert=False)
         distance_0 = superstructure(query_int_vectors[0], int_vectors[0])
         distance_1 = superstructure(query_int_vectors[0], int_vectors[1])
-        query, vecs = gen_query_vectors(binary_field_name, query_entities, default_top_k, nq, metric_type="SUPERSTRUCTURE")
+        query, vecs = gen_query_vectors(binary_field_name, query_entities, default_top_k, nq,
+                                        metric_type="SUPERSTRUCTURE")
         res = connect.search(binary_collection, query)
         assert len(res[0]) == 0
 
@@ -732,7 +743,8 @@ class TestSearchBase:
         top_k = 3
         int_vectors, entities, ids = init_binary_data(connect, binary_collection, nb=2)
         query_int_vectors, query_vecs = gen_binary_super_vectors(int_vectors, 2)
-        query, vecs = gen_query_vectors(binary_field_name, entities, top_k, nq, metric_type="SUPERSTRUCTURE", replace_vecs=query_vecs)
+        query, vecs = gen_query_vectors(binary_field_name, entities, top_k, nq, metric_type="SUPERSTRUCTURE",
+                                        replace_vecs=query_vecs)
         res = connect.search(binary_collection, query)
         assert len(res[0]) == 2
         assert len(res[1]) == 2
@@ -846,6 +858,24 @@ class TestSearchBase:
                 assert check_id_result(res[i], ids[i])
                 assert res[i]._distances[0] < epsilon
                 assert res[i]._distances[1] > epsilon
+
+    def _test_query_entities_with_field_less_than_top_k(self, connect, collection):
+        """
+        target: test search with field, and let return entities less than topk
+        method: insert entities and build ivf_ index, and search with field, n_probe=1
+        expected:
+        """
+        entities, ids = init_data(connect, collection)
+        simple_index = {"index_type": "IVF_SQ8", "params": {"nlist": 200}, "metric_type": "L2"}
+        connect.create_index(collection, field_name, simple_index)
+        default_query, default_query_vecs = gen_query_vectors(field_name, entities, default_top_k, nq, search_params={"nprobe": 1})
+        expr = {"must": [gen_default_vector_expr(default_query)]}
+        query = update_query_expr(default_query, expr=expr)
+        res = connect.search(collection, query, fields=["int64"])
+        for r in res[0]:
+            logging.getLogger().info(getattr(r.entity, "int64"))
+        assert len(res) == nq
+        assert len(res[0]) == default_top_k
 
 
 class TestSearchDSL(object):
@@ -991,7 +1021,8 @@ class TestSearchDSL(object):
         '''
         entities, ids = init_data(connect, collection)
         expr = {"must": [gen_default_vector_expr(default_query),
-                         gen_default_term_expr(values=[i for i in range(default_nb // 2, default_nb + default_nb // 2)])]}
+                         gen_default_term_expr(
+                             values=[i for i in range(default_nb // 2, default_nb + default_nb // 2)])]}
         query = update_query_expr(default_query, expr=expr)
         res = connect.search(collection, query)
         assert len(res) == nq
@@ -1007,7 +1038,8 @@ class TestSearchDSL(object):
         '''
         entities, ids = init_data(connect, collection)
         expr = {
-            "must": [gen_default_vector_expr(default_query), gen_default_term_expr(values=[1 for i in range(1, default_nb)])]}
+            "must": [gen_default_vector_expr(default_query),
+                     gen_default_term_expr(values=[1 for i in range(1, default_nb)])]}
         query = update_query_expr(default_query, expr=expr)
         res = connect.search(collection, query)
         assert len(res) == nq
@@ -1251,7 +1283,8 @@ class TestSearchDSL(object):
         '''
         entities, ids = init_data(connect, collection)
         term_first = gen_default_term_expr()
-        term_second = gen_default_term_expr(field="float", values=[float(i) for i in range(default_nb // 2, default_nb)])
+        term_second = gen_default_term_expr(field="float",
+                                            values=[float(i) for i in range(default_nb // 2, default_nb)])
         expr = {"must": [gen_default_vector_expr(default_query), term_first, term_second]}
         query = update_query_expr(default_query, expr=expr)
         res = connect.search(collection, query)
@@ -1586,7 +1619,8 @@ class TestSearchInvalid(object):
             pytest.skip("skip if index_type not matched")
         entities, ids = init_data(connect, collection)
         connect.create_index(collection, field_name, get_simple_index)
-        query, vecs = gen_query_vectors(field_name, entities, default_top_k, 1, search_params=search_params["search_params"])
+        query, vecs = gen_query_vectors(field_name, entities, default_top_k, 1,
+                                        search_params=search_params["search_params"])
         with pytest.raises(Exception) as e:
             res = connect.search(collection, query)
 
@@ -1601,8 +1635,10 @@ class TestSearchInvalid(object):
         index_type = "BIN_IVF_FLAT"
         int_vectors, entities, ids = init_binary_data(connect, binary_collection)
         query_int_vectors, query_entities, tmp_ids = init_binary_data(connect, binary_collection, nb=1, insert=False)
-        connect.create_index(binary_collection, binary_field_name, {"index_type": index_type, "metric_type": "JACCARD", "params": {"nlist": 128}})
-        query, vecs = gen_query_vectors(binary_field_name, query_entities, default_top_k, nq, search_params={"nprobe": 0}, metric_type="JACCARD")
+        connect.create_index(binary_collection, binary_field_name,
+                             {"index_type": index_type, "metric_type": "JACCARD", "params": {"nlist": 128}})
+        query, vecs = gen_query_vectors(binary_field_name, query_entities, default_top_k, nq,
+                                        search_params={"nprobe": 0}, metric_type="JACCARD")
         with pytest.raises(Exception) as e:
             res = connect.search(binary_collection, query)
 

--- a/tests/milvus_python_test/test_ping.py
+++ b/tests/milvus_python_test/test_ping.py
@@ -1,7 +1,7 @@
 import logging
 import pytest
 
-__version__ = '0.11.0'
+__version__ = '0.11.1'
 
 
 class TestPing:

